templates/common/on-prem/units/nodeip-configuration.service.yaml

```diff
@@ -23,6 +23,7 @@ contents: |
     --net=host \
     --security-opt label=disable \
     --volume /etc/systemd/system:/etc/systemd/system \
+    --volume /run:/run \
```

Comment:
This seems very broad. Have you considered having an ExecStartPre that creates /run/nodeip-configuration and mounting only that?

Reply:
optional , will change